src/Twitch.php

```diff
@@ -96,7 +96,7 @@ public function __construct()
             $this->setClientSecret($clientSecret);
         }
 
-        if ($redirectUri = config('twitch-api.client_secret')) {
+        if ($redirectUri = config('twitch-api.redirect_url')) {
             $this->setredirectUri($redirectUri);
```

Comment:
```suggestion
            $this->setRedirectUri($redirectUri);
```
Could you also fix the wrong method name?

Reply:
Done